guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,111 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/registering-slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+
+## Preparing commands
+
+To use autocomplete with your command options, you have to tell Discord to enable it.
+How to do this is dependent on how you create your slash commands.
+
+### Slash command builder
+
+The slash command builder does not yet support using autocomplete.
+
+### Custom system
+
+For other custom systems or entering it raw, your resulting code should look something like this. To enable autocomplete, set it to true in the option.
+
+```js {10}
+const commandData = {
+	name: 'autocomplete',
+	description: 'test command to show how autocomplete should be setup',
+	defaultPermission: true,
+	options: [
+		{
+			name: 'name',
+			description: 'Name of something',
+			type: 3, // String
+			autocomplete: true,
+		},
+	],
+};
```

Comment:
```suggestion
```js {9}
const commandData = new SlashCommandBuilder()
	.setName('autocomplete')
	.setDescription('test command to show how autocomplete should be setup')
	.setDefaultPermission(true)
	.addStringOption(opt =>
		opt
			.setName('name')
			.setDescription('Name of something')
			.setAutocomplete(true));
```